monitoring/grafana/dashboards/host-details.json

```diff
@@ -37,7 +37,7 @@
   "gnetId": null,
   "graphTooltip": 0,
   "id": null,
-  "iteration": 1557386759572,
+  "iteration": 1615564911000,
```

Comment:
@wornet-mwo Thank you for your contribution. Is this change really required?

Reply:
AFAIK this causes grafana to notify the client (browser) to refresh the dashboard. Maybe it isn't in fact but it's usually increased if a dashboard is edited in the frontend.